reports/module_od_to_ri_ec2.go

```diff
@@ -18,16 +18,102 @@ import (
 	"context"
 	"database/sql"
 	"errors"
-	"time"
-
+	"fmt"
+	"github.com/360EntSecGroup-Skylar/excelize"
 	"github.com/trackit/jsonlog"
-
 	"github.com/trackit/trackit/aws"
 	"github.com/trackit/trackit/aws/usageReports/history"
-	ec2 "github.com/trackit/trackit/onDemandToRI/ec2"
+	odRi "github.com/trackit/trackit/onDemandToRI/ec2"
 	"github.com/trackit/trackit/users"
+	"time"
 )
 
+const odToRiEc2UsageReportSheetName = "Od To RI EC2 Usage Report"
+
+var odToRiEc2UsageReportModule = module{
+	Name:          "Od To Ri EC2 Usage Report",
+	SheetName:     odToRiEc2UsageReportSheetName,
+	ErrorName:     "odToRiEc2UsageReportError",
+	GenerateSheet: generateOdToRiEc2UsageReportSheet,
+}
+
+// generateOdToRiEc2UsageReportSheet will generate a sheet with Od To Ri EC2 usage report
+// It will get data for given AWS account and for a given date
+func generateOdToRiEc2UsageReportSheet(ctx context.Context, aas []aws.AwsAccount, date time.Time, tx *sql.Tx, file *excelize.File) (err error) {
+	if date.IsZero() {
+		date, _ = history.GetHistoryDate()
+	}
+	fmt.Printf("launching generate sheet for od to ri ec2 usage report\n")
+	return OdToRiEc2UsageReportGenerateSheet(ctx, aas, date, tx, file)
+}
+
+func OdToRiEc2UsageReportGenerateSheet(ctx context.Context, aas []aws.AwsAccount, date time.Time, tx *sql.Tx, file *excelize.File) (err error) {
```

Comment:
@jgchoppe exported function in Go should have documentation

Reply:
@Dr0p42: don't review this PR it's still in progress for now